pkg/k8sutil/exec.go

```diff
@@ -36,28 +34,21 @@ var (
 	CephClusterNamespace string // Cephcluster namespace
 )
 
-func RunCommandInOperatorPod(cmd string, args []string, operatorNamespace, clusterNamespace string) {
-
-	// Get Kubernetes Client
-	rest, _, client := GetKubeClient()
+func RunCommandInOperatorPod(ctx *Context, cmd string, args []string, operatorNamespace, clusterNamespace string) {
```

Comment:
In Rook it's a more common pattern to use `context` as the variable name, and use `ctx` only when there is a conflict with a package name, or similar. So it would be nice to keep that same pattern and use `context` if we can.
```suggestion
func RunCommandInOperatorPod(context *Context, cmd string, args []string, operatorNamespace, clusterNamespace string) {
```

Reply:
here I think we'll have to use  `ctx` only but I'll follow the rook's approach where I can